cmd/litestream/main.go

```diff
@@ -903,6 +905,31 @@ func newNATSReplicaClientFromConfig(c *ReplicaConfig, _ *litestream.Replica) (_
 		return nil, fmt.Errorf("bucket required for NATS replica")
 	}
 
+	// Validate TLS configuration
+	// Both client cert and key must be specified together
+	if (c.ClientCert != "") != (c.ClientKey != "") {
+		return nil, fmt.Errorf("client-cert and client-key must both be specified for mutual TLS authentication")
+	}
+
+	// Validate that certificate files exist if specified
+	if c.ClientCert != "" {
+		if _, err := os.Stat(c.ClientCert); os.IsNotExist(err) {
+			return nil, fmt.Errorf("client certificate file not found: %s", c.ClientCert)
+		}
+	}
+	if c.ClientKey != "" {
+		if _, err := os.Stat(c.ClientKey); os.IsNotExist(err) {
+			return nil, fmt.Errorf("client key file not found: %s", c.ClientKey)
+		}
+	}
+
+	// Validate CA certificate files exist if specified
+	for _, caFile := range c.RootCAs {
+		if _, err := os.Stat(caFile); os.IsNotExist(err) {
+			return nil, fmt.Errorf("root CA certificate file not found: %s", caFile)
+		}
+	}
```

Comment:
This isn't necessary since the `nats.Connect` setup will catch and return errors if any of these files don't exist.